dataflow/src/main/java/org/checkerframework/dataflow/analysis/Analysis.java

```diff
@@ -400,13 +400,17 @@ protected void propagateStoresTo(
     /**
      * Updates the value of node {@code node} to the value of the {@code transferResult}. Returns
      * true if the node's value changed, or a store was updated.
+     *
+     * @param node a node
+     * @param transferResult the result type of a transfer function
```

Comment:
`the transfer result to use` maybe? The `type` is confusing, b/c we use the value from that transfer result.